pmlc/rt/opencl/opencl_wrappers.cc

```diff
@@ -11,115 +11,162 @@ namespace pmlc::rt::opencl {
 
 extern "C" {
 
-void *oclCreate(void *device) {
-  return new OpenCLInvocation(static_cast<OpenCLDevice *>(device));
+void *ocl_create_execenv(void *device) {
+  IVLOG(1, "ocl_create_execenv device=" << device);
+  void *result = new OpenCLInvocation(static_cast<OpenCLDevice *>(device));
+  IVLOG(1, "  ->" << result);
+  return result;
```

Comment:
I wonder if we should do a try/catch here and then come up with a way to report errors gracefully.

Reply:
Probably, but saving for the next PR at least.